test/utils_test.cpp

```diff
@@ -27,7 +28,20 @@ std::string getTmpFileName() {
   if (user != nullptr) {
     userstr = std::string(user);
   }
-  return std::string("/tmp/test_") + userstr + std::string(".graph");
+
+  // tmp dir
+  std::string tmpDir = "/tmp";
+  auto getTmpDir = [&tmpDir](const std::string& env) {
+    char* dir = std::getenv(env.c_str());
+    if (dir != nullptr) {
+      tmpDir = std::string(dir);
+    }
+  };
+  getTmpDir("TMPDIR");
+  getTmpDir("TEMP");
+  getTmpDir("TMP");
+
+  return tmpDir + "/test_" + userstr + ".graph";
```

Comment:
nit: does '/' work in windows ?

Reply:
Windows accepts both forward and back slashes as a separator since Windows XP.